prisma/schema.prisma

```diff
@@ -24,7 +24,7 @@ model Recipe {
   id Int @default(autoincrement()) @id
   name String
   description String
-  imageURL String? @map("image_url")
+  imageKey String? @map("image_key")
```

Comment:
In general, it's not a good way to handle files attached/related to database resources.

This setup introduces multiple problems:
- you lose original file name, mime-types etc of the files user uploads
- under the image key you do not save the full key with bucket so if your backend uses multiple buckets this will not work (or be problematic)
- if you want to add an upload feature to other resources you would have the same code/logic/keys all over the place. Hard to maintain